src/main/resources/db/master/changelog/12-modify-employee-column.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+  <changeSet id="12" author="nikhil">
+    <dropForeignKeyConstraint
+      baseTableName="leave_approval"
+      constraintName="fk_employee_leave_approval"/>
+    <modifyDataType
+      columnName="employee_id"
+      newDataType="varchar(100)"
+      tableName="employee"/>
+    <modifyDataType
+      columnName="approved_by_employee_id"
+      newDataType="varchar(100)"
+      tableName="leave_approval"/>
+    <addForeignKeyConstraint baseColumnNames="approved_by_employee_id"
+      baseTableName="leave_approval"
+      constraintName="fk_employee_leave_approval"
+      deferrable="true"
+      initiallyDeferred="true"
+      onDelete="CASCADE"
+      onUpdate="RESTRICT"
+      referencedColumnNames="employee_id"
+      referencedTableName="employee"/>
```

Comment:
This does not seem correct. 
`leave_application.employee_id` column should refer to `employee.id` column having data type integer and NOT varchar.
The same is the case with the below.
`leave_allocation.employee_id` column should refer to `employee.id` column having data type integer and NOT varchar.

Also, change` employee.employee_id` column to `employee.emp_id` column with type as `varchar`.